doc/tutorials/tutorial_X8.rst

```diff
@@ -57,6 +56,58 @@ For more details on configuring Strawberry Fields for cloud access, including
 using the command line interface, see the :doc:`/introduction/photonic_hardware`
 quickstart guide.
 
+Device details
+--------------
+
+Below, we will be submitting a job to run on the chip ``X8_01``.
+
+``X8_01`` is an 8-mode chip, with the following restrictions:
+
+* The initial states are two-mode squeezed states (:class:`~.S2gate`). We call modes 0 to 3 the
+  *signal modes* and modes 4 to 7 the *idler modes*. Two-mode squeezing is between the pairs
+  of modes: (0, 4), (1, 5), (2, 6), (3, 7).
+
+At this point, only the parameters :code:`r=1`, :code:`phi=0` are allowed for the two-mode
+squeezing gates between any pair of signal and idler modes. Eventually, a range of
+squeezing amplitudes :code:`r` will be supported.
+
+* Any arbitrary :math:`4\times 4` unitary (consisting of :class:`~.BSgate`, :class:`~.MZgate`,
+  :class:`~.Rgate`, and :class:`~.Interferometer` operations) can be applied identically
+  on both the signal and idler modes.
+
+* Finally, the chip terminates with photon-number resolving measurements (:class:`~.MeasureFock`).
+
+.. note::
+
+    **What's in a name?**
+
+    The name of the photonic chip, ``X8_01``, provides several clues to the underlying
+    chip structure and architecture.
+
+    At Xanadu, photonic devices are named using the nomenclature ``AM_YY``, where:
+
+    * ``M`` is the **mode number**, the number of modes available on the chip. ``X8_01``
+      has 8 available modes.
+
+    * ``A`` is the **architecture number.** This chip belongs to the ``X``-series of photonic
+      devices. Devices in this series are characterized by the following properties:
+
+      - The available modes :math:`M` are divided into two main groups, signal modes
+        (numbered :math:`[0, M/2-1]`) and idler modes (numbered :math:`[M/2, M-1]`).
+
+      - Initial two-mode squeezed states entangle the signal modes and the idler modes
+        in a 'cascade'; entangled mode pairs are :math:`[(0, M/2), (1, M/2+1), \dots, (M/2-1, M-1)]`.
+
+      - An arbitrary :math:`M/2\times M/2` unitary can then be applied to both the signal and idler modes.
+
+      - Finally, the modes terminate with photon-number resolving detectors.
+
+    * ``YY`` is the **chip ID**. It specifies a particular chip instance chosen for executing the jobs.
+      While all chips in the same series have the same architecture, specific chips may be characterized
+      by hardware imperfections, including loss and gate fidelity.
+
+    While ``X8`` is used here to specify *any* chip in the ``X8``-series (including ``X8_01``), when
+    submitting remote jobs, a specific chip ID must be chosen.
```

Comment:
@lneuhaus and @trbromley, have added in a note box here!

Reply:
(I defined mode number before architecture number, because it is useful to have the number of modes already defined)

Reply:
This is a cool box! I could see it being quite a popular bit of information.

One thing I wasn't sure on is whether we will really change from `X` when we go to other types of chip? E.g., the X series is strictly for two-mode squeezing chips? I'd potentially leave out the architecture number bullet point so as not to over specify at this stage. Although, if this rule has already been fixed, then I'm happy for it to go here.

Reply:
> One thing I wasn't sure on is whether we will really change from X when we go to other types of chip? E.g., the X series is strictly for two-mode squeezing chips? I'd potentially leave out the architecture number bullet point so as not to over specify at this stage. Although, if this rule has already been fixed, then I'm happy for it to go here.

I agree this is not written in stone yet. Maybe @xanadu-zvernon has a more definite opinion on this?

Reply:
This is correct as written above, so no need for change.